### DISCUSSION TITLE
Error in deconvolute_cibersort

### DISCUSSION THREAD

**Discussion by huangwb8** [omnideconv/immunedeconv#65] [Q&A] [answered]
When running `deconvolute_cibersort`, some errors appear:
```
assertion failed: CIBERSORT.R is provided
error: exists("cibersort_binary", envir = config_env) is not TRUE
```
It seems that there is no function `cibersort_binary` loaded. Could you please repair this bug? Thanks!

**Comment by grst** [comment, marked as answer]
Hi @huangwb8, 

you'll need to provide the path to the `CIBERSORT.R` file as specified here: 
https://omnideconv.org/immunedeconv/articles/immunedeconv.html#special-case-cibersort

**Comment by huangwb8** [comment]
Thanks! I got it!

**Comment by Minh-AnhHuynh** [comment]
Just for reference for anyone having the same error. Here is the new website in case anyone is looking for it :
https://omnideconv.org/immunedeconv/articles/immunedeconv.html#special-case-cibersort
